index.html

```diff
@@ -75,6 +75,7 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
                              is given
 </pre>
 <p>NOTE: The message type is the name of the event type to send without the suffix <em>event</em> and in small case letters, for example <em>eiffelactivityfinished</em> for an <em>EiffelActivityFinishedEvent</em>.</p>
+<p>For other than eiffelsemantics, we need to provide the <strong><em>-Djava.ext.dirs="Path to other protocol jars"</em></strong> for loading other protocol jars.</p>
```

Comment:
Maybe, it would be better to rewrite the sentence as:
For other messaging protocol than "eiffelsemantics", ...